cfme/tests/ansible/test_embedded_ansible_basic.py

```diff
@@ -457,6 +457,26 @@ def test_embedded_ansible_repository_playbook_link(ansible_repository):
     assert view.is_displayed
 
 
+@pytest.mark.tier(2)
+@pytest.mark.meta(automates=[1734446])
+def test_embedded_ansible_repository_playbook_sub_dir(ansible_repository):
+    """
+    The Embedded Ansible role should find playbooks in sub folders.
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: Ansible
+        initialEstimate: 1/6h
+        tags: ansible_embed
+    """
+    # Navigation to playbook view from Ansible Repository
+    playbook_view = navigate_to(ansible_repository, "Playbooks")
+    # 'more_playbooks/hello_world.yml' Playbook name is hardcoded in order to find specific
+    # playbook in the Ansible testing repository.
+    # Asserting the playbook is available.
+    assert playbook_view.entities.get_entity(name="more_playbooks/hello_world.yml", surf_pages=True)
```

Comment:
Optional: @sbulage like this but 
1. If an entity not found then it will raise `ItemNotFound` Error, I don't think we need `assert` in such case.
2. we can just check entity in `all_entity_names`.

Reply:
@digitronik I have updated with suggested changes.